### PR TITLE
code-server: add pnpm support

### PR DESCRIPTION
**Pull request by phyesix**

### CODE REVIEW

**Comment by phyesix** [comment]
When I enter the following command, 

`find ./ -path "./.git" -prune -o ( -name "run" -o -name "finish" -o -name "check" ) -not -perm -u=x,g=x,o=x -print -exec chmod +x {} + `

I get an error like in the image:

Screenshot 2024-02-27 at 20 49 44

**Comment by thespad** [comment]
The GHA output is stripping the `\` that should be there.
```
find ./ -path "./.git" -prune -o \( -name "run" -o -name "finish" -o -name "check" \) -not -perm -u=x,g=x,o=x -print -exec chmod +x {} +
```

File: README.md [hunk 1]
```diff
@@ -1,25 +1,7 @@
-# Rsync - Docker mod for openssh-server
+# pnpm - Docker mod for code-server
 
-This mod adds rsync to openssh-server, to be installed/updated during container start.
+This mod adds [PNPM](https://github.com/pnpm/pnpm) to [linuxserver/code-server](https://github.com/linuxserver/docker-code-server), to be installed during container start.
```
Comment (aptalca) [inline README.md:3]: Only for code-server? Why not both like [in here](https://github.com/linuxserver/docker-mods/tree/code-server-flutter?tab=readme-ov-file#flutter-beta---docker-mod-for-code-server-and-openvscode-server)?

File: root/etc/s6-overlay/s6-rc.d/init-mod-code-server-pnpm-install/run [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/with-contenv bash
+
+echo "**** installing pnpm  ****"
+wget -qO- https://get.pnpm.io/install.sh | ENV="$HOME/.bashrc" SHELL="$(which bash)" bash -
```
Comment (aptalca) [inline root/etc/s6-overlay/s6-rc.d/init-mod-code-server-pnpm-install/run:4]: There is no wget in the image

**Pull request closed without merging**